man/src/config.1.md

```diff
@@ -0,0 +1,196 @@
+: Move window
+
+Focus **args** [MOVEMENT]
+: Move Focus
+
+Quit
+: Quit the window manager
+
+Kill
+: Kill the currently focused window
+
+Restart
+: Reloads the config and restarts components
+
+Layout **args** [LAYOUT]
+: Change the current layout
+
+GoToWorkspace **args** [WORKSPACE_ARGS]
+: Change the current workspace
+
+MoveToWorkspace **args** [WORKSPACE_ARGS]
+: Move the used window to a different workspace
+
+MoveToWorkspaceAndFollow **args** [WORKSPACE_ARGS]
+: Move the focused window to and select a different workspace
+
+Exec **args** <COMMAND>
+: Execute a given command
+
+Fullscreen
+: Toggle fullscreen mode for the focused window
+
+## ARGS
+
+### MOVEMENT:
+
+Left
+: Moves to the left
+
+right
+: Moves to the right
+
+## LAYOUT
+
+VerticalStriped
+: windows vertically next to each other
+
+HorizontalStriped
+: windows a horizontally underneath each other
+
+None
+: if no argument is provided, the next layout is chosen
+
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
+
+Index
+: workspace with the given index
+
+## ITERATIONS
+
+iter
+: iteraates over given number in order to change
+
+# DEFAULT KEYBINDINGS
+
+TODO: DEFAULT KEYBINDINGS
+
+## EXAMPLES
+
+TODO: SAMPLE CONFIG
+
+### KEYBINDINGS
+
+```yaml
+cmds:
+  - keys: ["M", "t"]
+    commands:
+      - command: Exec
+        args: "firefox"
+```
+
+In this example pressing the meta key and 't', a new firefox window is opened.
+
+### ITERATIONS
+
+```yaml
+iter_cmds:
+  - iter: [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    command:
+      keys: ["A", "C", "$VAR"]
+      commands:
+        - command: GoToWorkspace
+          args: "$VAR"
+```
+
+In this example using the ALT and CONTROLL key paired with a number from one to nine, the user can go to the desired workspace.
+`$VAR` is a reference for the entered iterator.
+
+## BORDERS
+
+border_witdh
+: sets the border witdh of windows in
+
+border_color
+: sets the border color and has to be entered in hexadecimal
+
+border_focus_color
+: sets the border color for focused nbdows and has to be entered in hexadecimal
+
+gap
+: gap between windows in pixels
+
+## EXECTUE
+
+exec
+: onetime execution when the window manager starts
+
+exec_always
+: executes when the windosw manager starts or restarts
+
+# Bugs
+
+Please open an issue <https://github.com/DHBW-FN/OxideWM/issues> .
+
+# COPYRIGHT
+
+Copyright © 2023 Philipp Kalinowski GPLv3+\: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
+This is free software: You are free to change and redistribute it. There is NO WARRANTY to the extent permitted by law.
+
+# FURTHER DOCUMENTATION
+
+Access the full Oxide documentation under **https://oxide.readthedocs.io/**.
+
+# SEE ALSO
+
+**oxide(1)**, **oxide-msg**, **oxide-bar(1)**
```

Comment:
**oxide(1)**, **oxide-msg(1)**, **oxide-bar(1)**

The one is important as this labels our man pages as part of the user space program man page section